doc/source/rllib-algorithms.rst

```diff
@@ -406,6 +407,26 @@ HalfCheetah    13000       ~15000
    :start-after: __sphinx_doc_begin__
    :end-before: __sphinx_doc_end__
 
+.. _maml:
+
+Model-Agnostic Meta-Learning (MAML)
```

Comment:
Can you add a link from this section too? https://docs.ray.io/en/latest/rllib-toc.html#algorithms